subprojects/docs/src/docs/userguide/embedding.adoc

```diff
@@ -66,10 +66,73 @@ The main entry point to the Tooling API is the api:org.gradle.tooling.GradleConn
 [[sec:embedding_compatibility]]
 === Gradle version and Java version compatibility
 
-The current version of the Tooling API supports running builds using Gradle versions 1.2 and later.
+==== Provider side
```

Comment:
⭕️ I'd be in favor of not adding so much more text. How about just adding a second sentence after the one that was there before?

"Running builds with Gradle versions older than 2.6 is deprecated and support will be removed in the Tooling API version 5.0"